test/yargs.js

```diff
@@ -110,6 +110,18 @@ describe('yargs dsl tests', () => {
     argv.looks.should.eql('good')
   })
 
+  it('should ignore a missing array choice with an empty default', () => {
```

Comment:
In yargs/yargs-parser#205, the bug seems to depend on the use of `flatten-duplicate-arrays` or not. Souldn't this test reproduce this?